enhancements/local-storage/automatic-detection.md

```diff
@@ -0,0 +1,460 @@
+---
+title: automatic-detection-of-disks-in-Local-Storage-Operator
+authors:
+  - "@ashishranjan738"
+  - "@rohantmp"
+  - "@jarrpa"
+reviewers:
+  - "@jsafrane"
+  - "@hekumar"
+  - "@chuffman"
+  - "@sapillai"
+  - "@leseb"
+  - "@travisn"
+  - "@rohantmp"
+  - "@jarrpa"
+approvers:
+  - "@jsafrane"
+  - "@hekumar"
+  - "@chuffman"
+creation-date: 2020-01-21
+last-updated: 2020-03-12
+status: implementable
+---
+
+# Automatic detection of Disks in Local-Storage-Operator 
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This is a proposal to enable automated discovery of local devices via the `VolumeDiscovery` CRD and sorting them into StorageClasses based on their characteristics via the `AutoDetectVolume` CRD. These CRDs will be implemented in the LocalStorageOperator(LSO)
```

Comment:
local devices -> local disk devices. It won't detect GPUs and similar stuff.

Reply:
done